src/olympia/reviewers/serializers.py

```diff
@@ -203,3 +203,14 @@ def get_validation_url(self, obj):
 
     def get_has_been_validated(self, obj):
         return obj.current_file.has_been_validated
+
+
+class DiffableVersionSerializer(VersionSerializer):
+    should_show_channel = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Version
+        fields = ('id', 'channel', 'url', 'version', 'should_show_channel')
+
+    def get_should_show_channel(self, obj):
+        return self.context.get('should_show_channel', False)
```

Comment:
So we return the same value for every version in the list response?

Reply:
Correct, I didn't want to nest the list of versions tbh so I decided to list that for every version.

On the other hand, we're returning the `channel` for every version so the client might be able to figure this out themselves? cc @willdurand (some context: If we have mixed listed/unlisted versions that can be compared we want to show the channel, otherwise we don't need to)

Reply:
Thanks. Could you maybe add this note about channels in the API doc?

Reply:
Added in 3801b2d